fbgemm_gpu/bench/split_table_batched_embeddings_benchmark.py

```diff
@@ -797,8 +797,10 @@ def uvm(
             offsets = torch.tensor(([0] + np.cumsum(lengths).tolist())).int().cuda()
             per_sample_weights = None
             if weighted:
-                assert (this_rs_uvm_weights := rs_uvm[2]) is not None
-                assert (this_rs_gpu_weights := rs_gpu[2]) is not None
+                this_rs_uvm_weights = rs_uvm[2]
+                assert this_rs_uvm_weights is not None
+                this_rs_gpu_weights = rs_gpu[2]
+                assert this_rs_gpu_weights is not None
```

Comment:
Could you explain why this change is needed for ROCm?

Reply:
> Could you explain why this change is needed for ROCm?

ROCm is on Python 3.7.

Reply:
Maybe I just don't understand Python, but
- Does ROCm depend on a specific Python version? In my understanding, it is irrelevant.
- Does the original grammar depend on a specific Python version (e.g., only Python2, or Python 3.8+, or Python <3.5).
```py
assert (this_rs_uvm_weights := rs_uvm[2]) is not None
```

Reply:
":=" is a Python 3.8 feature, see https://stackoverflow.com/a/26000366
The PyTorch upstream CI jobs for ROCm are executed with python 3.7, so all our release dockers are on 3.7.

Reply:
Yes maybe we should change this if FBGEMM_GPU should work with Python < 3.8,, but I am not sure about the minimum Python version for FBGEMM_GPU, but @jianyuh do you know the version requirement?

Reply:
Thanks! If the current change works for 3.7+, it looks good to me. Ideally we should use the simple syntax and avoid using the specific Python 3.8+ features.